### PR TITLE
Added the creation of a .gitgnore file for studio courses

### LINKED ISSUE #37: Add .gitignore file to studio export repos

### PR DESCRIPTION
**Pull request by carsongee**
Closes #37 

@itsbenweeks 
I tested with https://github.mit.edu/MITx-Studio2LMS/content-mit-DevOooooop002-Spring_2090 and it worked for me, but I would love if you could test as well.  It should work with the documented perms we already require.

### CODE REVIEW

File: orcoursetrion/config.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
     'ORC_GH_API_URL': 'https://github.mit.edu/api/v3/',
 
     # GitHub Committer Name
-    'ORC_GH_NAME': 'Orcoursetrion ODL',
+    'ORC_GH_NAME': 'Orcoursetrion',
 
     # GitHub Committer Email
     'ORC_GH_EMAIL': 'orcoursetrion@example.com',
```
Comment (itsbenweeks) [inline orcoursetrion/config.py]: is there a way we can get these from a git config file?

Comment (carsongee) [inline orcoursetrion/config.py]: That makes it difficult to test and a bit less flexible, you can configure them with environment variables though

Comment (itsbenweeks) [inline orcoursetrion/config.py]: Makes sense. I'll add that to the docs, then.

**Comment by itsbenweeks** [comment]
:neckbeard: 
:necktie: :+1: 
:jeans: 
Neckbeard approves.

**Pull request merged**